Proposals/NNNN-attributedstring-scope-enumeration.md

```diff
@@ -0,0 +1,93 @@
+# `AttributedString` Scope Enumeration
+
+* Proposal: [SF-NNNN](NNNN-attributedstring-scope-enumeration.md)
+* Authors: [Jeremy Schonfeld](https://github.com/jmschonfeld)
+* Review Manager: TBD
+* Implementation: [swiftlang/swift-foundation#1093](https://github.com/swiftlang/swift-foundation/pull/1093)
+* Status: **Pitch**
```

Comment:
```suggestion
* Status: **Accepted**
* Review: **[Pitch](https://forums.swift.org/t/pitch-attributedstring-scope-enumeration/76573)**
```

(I realized we never really required a Review section in the proposal document, but I think it's nice to have so we can reference back to the discussion thread if needed)